bundles/auth/org.eclipse.smarthome.auth.oauth2client.test/OSGI-INF/metatype/org.eclipse.smarthome.auth.oauth2client.test.AuthorizationCodeTestAgent.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.osgi.org/xmlns/metatype/v1.1.0 http://www.osgi.org/xmlns/metatype/v1.1.0">
+	<OCD name="AuthorizationCodeTestAgent configurations" id="org.eclipse.smarthome.auth.oauth2client.test.AuthorizationCodeTestAgent"
+		description="OAuth AuthorizationCode TestAgent configurations">
+
+		<AD name="TOKEN_URL" id="TOKEN_URL" default="https://blah/oauth2/token" required="true" type="String"
+			description="OAuth provider Token URL" />
+		<AD name="AUTH_URL" id="AUTH_URL" default="https://blah/oauth2/authorize" required="true"
+			type="String" description="OAuth provider Authorization Code URL" />
+		<AD name="CLIENT_ID" id="CLIENT_ID" default="myAwsomeAppId" required="true" type="String"
+			description="The client application id" />
+		<AD name="CLIENT_SECRET" id="CLIENT_SECRET" default="myAwsomeAppSecret" required="true" type="String"
+			description="The client application secret" />
+
+		<AD name="REDIRECT_URI" id="REDIRECT_URI" default="https://myAwsomeApp/oauth2/redirected" required="true"
+			type="String" description="Redirect URI" />
+
+		<AD name="USERNAME" id="USERNAME" default="not used" required="true" type="String" description="Not used in this agent" />
+		<AD name="PASSWORD" id="PASSWORD" default="not used" required="true" type="String" description="Not used in this agent" />
+		<AD name="SCOPE" id="SCOPE"
+			default="scope1 scope2 scope3"
```

Comment:
If you intend to list and select multiple values, you can use `Option` xml tag

Reply:
Cannot.  This is meant to be a space delimited list, with open-ended options defined by oauth provider.